### PR TITLE
create dapr install dir before copy

### LINKED ISSUE #1111: dapr install script
## Expected Behavior

install successfully to `/usr/local/bin` on centos7

## Actual Behavior

install failed on centos7

## Steps to Reproduce the Problem

wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh 1.7.1 -O - | /bin/bash

sh install.sh 1.7.1
Your system is linux_amd64
Installing Dapr CLI...

Installing v1.7.1 Dapr CLI...
Downloading https://github.com/dapr/cli/releases/download/v1.7.1/dapr_linux_amd64.tar.gz ...
Failed to install dapr
Failed to install Dapr CLI
For support, go to https://dapr.io

## Problem

The target dapr cli binary save directory `/usr/local/bin` may not exist by default, it need make sure that dir does exist before copying tar extraced binary file

### PR DESCRIPTION
**Pull request by hdget**
# Description

dapr install failed on centos7, by default, the default dapr cli binary install dir: `/usr/local/bin` may not exist on some linux distribution, it need make sure such dir does exist before cp cli binary to there

## Issue reference

closes #1111

### CODE REVIEW

**Comment by shubham1172** [comment]
@hdget please fix the DCO.

File: install/install.sh [hunk 1]
```diff
@@ -180,6 +180,7 @@ installFile() {
         runAsRoot rm "$DAPR_CLI_FILE"
     fi
     chmod o+x $tmp_root_dapr_cli
+    mkdir -p $DAPR_INSTALL_DIR
```
Comment (mukundansundar) [inline install/install.sh:183]: Check if it does not exist and then create it please

**Review by mukundansundar — CHANGES_REQUESTED** [review]

**Comment by mukundansundar** [comment]
@hdget thanks for the contribution. One small additional check and also please fix the DCO

File: install/install.sh (hunk 1 above)
Comment (shubham1172) [reply install/install.sh:183]: Doesn't `mkdir -p` already take care of it?

**Review by mukundansundar — CHANGES_REQUESTED** [review]
@hdget Please signoff on the commit ... https://github.com/dapr/cli/blob/master/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work

**Review by shubham1172 — APPROVED** [review]

**Comment by hdget** [comment]
@mukundansundar @shubham1172 fossa/fossa-scan step failed.

**Comment by shubham1172** [comment]
> @mukundansundar @shubham1172 fossa/fossa-scan step failed.

That should be okay, it's due to an ongoing bug from FOSSA.

**Comment by shubham1172** [comment]
@mukundansundar can you please merge this if it looks good to you?

**Comment by shubham1172** [comment]
@mukundansundar ping

**Review by pravinpushkar — APPROVED** [review]

**Comment by mukundansundar** [comment]
This will be merged after 1.10 release cut ...

**Review by mukundansundar — APPROVED** [review]

**Comment by shubham1172** [comment]
Can we merge this now @mukundansundar?

**Comment by yaron2** [comment]
Thanks for this contribution @hdget, apologies it's taken so long to merge.

**Pull request merged**